modules/apps/sharing/sharing-web/src/main/java/com/liferay/sharing/web/internal/display/context/ViewSharedAssetsDisplayContext.java

```diff
@@ -304,6 +308,17 @@ public boolean isSharingEntryVisible(SharingEntry sharingEntry)
 		return true;
 	}
 
+	private SharingEntryDropdownItemContributor
+			_getSharingEntryMenuItemContributor(
+				ClassNameLocalService classNameLocalService, long classNameId)
```

Comment:
Hey @Valmir-unicap, the `classNameLocalService` is an instance field and do not need to be passed as parameter.